src/main/java/rx/internal/operators/OperatorOnBackpressureBuffer.java

```
@@ -156,7 +157,15 @@ private boolean assertCapacity() {
                                 "Overflowed buffer of "
                                         + baseCapacity));
                         if (onOverflow != null) {
-                            onOverflow.call();
+                            try {
+                                onOverflow.call();
+                            } catch (Throwable e) {
+                                Exceptions.throwIfFatal(e);
```

Need to notify the plugin instead of swallowing the non fatal exceptions.